pkg/cmd/roachprod/main.go

```diff
@@ -885,6 +885,47 @@ var getProvidersCmd = &cobra.Command{
 	},
 }
 
+var grafanaStartCmd = &cobra.Command{
+	Use:   `grafana-start <cluster> [<grafanaConfig>]`,
+	Short: `spins up a prometheus and grafana instances on the last node provided`,
+	Long: `spins up a prometheus and grafana instances on the highest numbered node in the cluster
+and will scrape from all nodes in the cluster`,
+	Args: cobra.ExactArgs(1),
+	Run: wrap(func(cmd *cobra.Command, args []string) error {
+		return roachprod.StartGrafana(context.Background(), roachprodLibraryLogger, args[0],
+			grafanaConfig, nil)
+	}),
+}
+
+var grafanaStopCmd = &cobra.Command{
+	Use:   `grafana-stop <cluster>`,
+	Short: `spins down prometheus and grafana instances on the last node in the cluster`,
+	Long: `spins down the prometheus and grafana instances on provided roachprod node and
+dumps the prometheus data at the root of the last node in the cluster`,
```

Comment:
Omit the "dumps the prometheus data at the root of the last node in the cluster" sentence here, it's already present in the help text for the command. Also, how do you actually make use of this dump?